### DISCUSSION TITLE
How to not get redirect to the home page using nuxt 3?

### DISCUSSION THREAD

**Discussion by Guito** [vite-pwa/vite-plugin-pwa#559] [Q&A]
Hello everybody,

I am having the same problem as listed here https://github.com/vite-pwa/docs/issues/58

When accessing any other page which is not the default one I am redirected to the default one. Here it is my config:

```
 pwa: {
        registerType: 'autoUpdate',
        manifest: {
           ...
        },
        workbox: {
            navigateFallback: null,
            globPatterns: ['**/*.{js,css,html,png,svg,ico}']
        },
        client: {
            installPrompt: true,
        },
        devOptions: {
            enabled: true,
            type: 'module'
        },
    },
```

I tried several things but I could not achieve the correct behaviour.

**Comment by userquin** [comment]
@Guito check https://github.com/vite-pwa/docs/issues/58#issuecomment-1704155230
